### ISSUE TITLE
Exception when disposing HandSignatureControl

### ISSUE THREAD

**Issue by thirteenthstep** [RomanBase/hand_signature#5]
Disposing the control leads to an exception, probably since listeners are being notified after 'super.dispose()' gets called:

````
════════ Exception caught by widgets library ═══════════════════════════════════════════════════════
The following assertion was thrown while finalizing the widget tree:
A HandSignatureControl was used after being disposed.

Once you have called dispose() on a HandSignatureControl, it can no longer be used.
When the exception was thrown, this was the stack: 
#0      ChangeNotifier._debugAssertNotDisposed.<anonymous closure> (package:flutter/src/foundation/change_notifier.dart:106:9)
#1      ChangeNotifier._debugAssertNotDisposed (package:flutter/src/foundation/change_notifier.dart:112:6)
#2      ChangeNotifier.notifyListeners (package:flutter/src/foundation/change_notifier.dart:201:12)
#3      HandSignatureControl.clear (package:hand_signature/src/signature_control.dart:814:5)
#4      HandSignatureControl.dispose (package:hand_signature/src/signature_control.dart:1125:5)
...
════════════════════════════════════════════════════════════════════════════════════════════════════

**Comment by RomanBase** [comment]
You are right, `notifyListeners` is used after `dispose`.
Will be fixed in next minor version.

Thanks for notice..

**Comment by RomanBase** [comment]
Solved in version 0.6.3
